pkg/operator/ceph/csi/csi.go

```diff
@@ -94,6 +94,7 @@ func (r *ReconcileCSI) setParams() error {
 	CSIParam.SnapshotterImage = k8sutil.GetValue(r.opConfig.Parameters, "ROOK_CSI_SNAPSHOTTER_IMAGE", DefaultSnapshotterImage)
 	CSIParam.KubeletDirPath = k8sutil.GetValue(r.opConfig.Parameters, "ROOK_CSI_KUBELET_DIR_PATH", DefaultKubeletDirPath)
 	CSIParam.VolumeReplicationImage = k8sutil.GetValue(r.opConfig.Parameters, "CSI_VOLUME_REPLICATION_IMAGE", DefaultVolumeReplicationImage)
+	CSIParam.CSIAddonsImage = k8sutil.GetValue(r.opConfig.Parameters, "CSI_ADDONS_IMAGE", DefaultCSIAddonsImage)
```

Comment:
Prefix with "ROOK_" to follow the existing trend.

```suggestion
	CSIParam.CSIAddonsImage = k8sutil.GetValue(r.opConfig.Parameters, "ROOK_CSI_ADDONS_IMAGE", DefaultCSIAddonsImage)
```

Reply:
done